tests/test_obs_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import unittest
+
+from lsst.utils import getPackageDir
+import lsst.afw.geom
+import lsst.obs.base.tests
+import lsst.obs.test
+
+
+class TestObsTest(lsst.obs.base.tests.ObsTests, unittest.TestCase):
+    def setUp(self):
+        product_dir = getPackageDir('obs_test')
+        self.data_dir = os.path.join(product_dir, 'data', 'input')
+        self.Mapper = lsst.obs.test.TestMapper
+
+        super(TestObsTest, self).setUp()
+
+        self.dataIds = {'raw': {'visit': 1, 'filter': 'g'},
+                        'bias': {'visit': 1},
+                        'flat': {'visit': 1},
+                        '1': {'visit': 1, 'filter': 'g'},
+                        '2': {'visit': 2, 'filter': 'g'},
+                        '3': {'visit': 3, 'filter': 'r'}
+                        }
+        self.path_to_raw = os.path.join(self.data_dir, "raw", "raw_v1_fg.fits.gz")
+        self.result_ids = {'raw': 1, 'bias': 1, 'flat': 1}
```

Comment:
I find this name confusing; what kind of IDs does this contain? If it's a visit ID, then `result_visits` would probably be clearer